[2024-05-30T23:49:20.826-0500] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-30T23:49:20.852-0500] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskB manual__2024-05-31T04:49:15.633241+00:00 [queued]>
[2024-05-30T23:49:20.859-0500] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskB manual__2024-05-31T04:49:15.633241+00:00 [queued]>
[2024-05-30T23:49:20.860-0500] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-30T23:49:20.880-0500] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): taskB> on 2024-05-31 04:49:15.633241+00:00
[2024-05-30T23:49:20.883-0500] {standard_task_runner.py:63} INFO - Started process 9983 to run task
[2024-05-30T23:49:20.886-0500] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'executing_multiple_tasks', 'taskB', 'manual__2024-05-31T04:49:15.633241+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/execute_multiple_tasks.py', '--cfg-path', '/tmp/tmpcmebv5nq']
[2024-05-30T23:49:20.888-0500] {standard_task_runner.py:91} INFO - Job 8: Subtask taskB
[2024-05-30T23:49:20.937-0500] {task_command.py:426} INFO - Running <TaskInstance: executing_multiple_tasks.taskB manual__2024-05-31T04:49:15.633241+00:00 [running]> on host diallo
[2024-05-30T23:49:21.071-0500] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Diallo' AIRFLOW_CTX_DAG_ID='executing_multiple_tasks' AIRFLOW_CTX_TASK_ID='taskB' AIRFLOW_CTX_EXECUTION_DATE='2024-05-31T04:49:15.633241+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-31T04:49:15.633241+00:00'
[2024-05-30T23:49:21.072-0500] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-30T23:49:21.090-0500] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-30T23:49:21.092-0500] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK B has executed']
[2024-05-30T23:49:21.099-0500] {subprocess.py:86} INFO - Output:
[2024-05-30T23:49:21.100-0500] {subprocess.py:93} INFO - TASK B has executed
[2024-05-30T23:49:21.100-0500] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-30T23:49:21.101-0500] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-30T23:49:21.123-0500] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=executing_multiple_tasks, task_id=taskB, run_id=manual__2024-05-31T04:49:15.633241+00:00, execution_date=20240531T044915, start_date=20240531T044920, end_date=20240531T044921
[2024-05-30T23:49:21.178-0500] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-30T23:49:21.193-0500] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-30T23:49:21.193-0500] {local_task_job_runner.py:222} INFO - ::endgroup::
